To: vim_dev@googlegroups.com
Subject: Patch 7.3.079
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.079
Problem:    Duplicate lines in makefile.
Solution:   Remove the lines. (Hong Xu)
Files:	    src/Make_mvc.mak


*** ../vim-7.3.078/src/Make_mvc.mak	2010-11-03 21:59:23.000000000 +0100
--- src/Make_mvc.mak	2010-12-08 14:53:16.000000000 +0100
***************
*** 380,388 ****
  !if "$(_NMAKE_VER)" == "10.00.30319.01"
  MSVCVER = 10.0
  !endif
- !if "$(_NMAKE_VER)" == "9.00.30729.01"
- MSVCVER = 9.0
- !endif
  !endif
  
  # Abort bulding VIM if version of VC is unrecognised.
--- 380,385 ----
*** ../vim-7.3.078/src/version.c	2010-12-08 14:23:08.000000000 +0100
--- src/version.c	2010-12-08 14:54:02.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     79,
  /**/

-- 
login: yes
password: I don't know, please tell me
password is incorrect
login: yes
password: incorrect

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
